packages/amplify-graphql-api-construct-tests/src/sql-datatabase-controller.ts

```diff
@@ -285,14 +288,20 @@ export class SqlDatatabaseController {
         )}`,
       );
     }
+
     const detailsStr = JSON.stringify({
       dbConfig: this.databaseDetails.dbConfig,
       dbConnectionConfig,
-      schemaConfig: schemaConfigString,
     });
-    const filePath = path.join(projRoot, 'db-details.json');
-    fs.writeFileSync(filePath, detailsStr);
-    console.log(`Wrote DB details at ${filePath}`);
+    const stackConfigStr = JSON.stringify(stackConfig ?? {});
+
+    const dbDetailsfilePath = path.join(projRoot, 'db-details.json');
+    fs.writeFileSync(dbDetailsfilePath, detailsStr);
+    console.log(`Wrote DB details at ${dbDetailsfilePath}`);
+
+    const stackConfigFilePath = path.join(projRoot, 'stack-config.json');
+    fs.writeFileSync(stackConfigFilePath, stackConfigStr);
+    console.log(`Wrote stack config to ${stackConfigFilePath}`);
```

Comment:
If `stackConfigStr`  is `{}` perhaps there's no need to write anything and that could be logged instead.